slice/IceGrid/Descriptor.ice

```diff
@@ -102,6 +102,27 @@ module IceGrid
     ["java:type:java.util.LinkedList<AdapterDescriptor>"]
     sequence<AdapterDescriptor> AdapterDescriptorSeq;
 
+    /// A Freeze database environment descriptor (deprecated, no longer used).
+    ["deprecated:This descriptor is provided for schema compatibility. It is no longer used as of Ice 3.8."]
+    struct DbEnvDescriptor
+    {
+        /// The name of the database environment.
+        string name;
+
+        /// The description of this database environment.
+        string description;
+
+        /// The home of the database environment.
+        string dbHome;
+
+        /// The configuration properties of the database environment.
+        PropertyDescriptorSeq properties;
+    }
+
+    /// A sequence of {@link DbEnvDescriptor}.
+    ["deprecated:This descriptor is provided for schema compatibility. It is no longer used as of Ice 3.8."]
+    sequence<DbEnvDescriptor> DbEnvDescriptorSeq;
```

Comment:
Why not keep the `["java:type:java.util.LinkedList<DbEnvDescriptor>"]` metadata that used to be on this?